source/core/experimental/stdcpp/typeinfo.d

```diff
@@ -143,5 +142,144 @@ else version (CppRuntime_Gcc)
         override const(char)* what() const;
     }
 }
+else version (CppRuntime_Clang)
+{
+    extern (C++, "std"):
+
+    static assert(__traits(classInstanceSize, type_info)  == 16);
+    static assert(__traits(classInstanceSize, bad_cast)   ==  8);
+    static assert(__traits(classInstanceSize, bad_typeid) ==  8);
+
+    class type_info
+    {
+        ~this() {}
+
+    final:
+        this(const ref type_info);
+        // identity assignment operator overload is illegal
+        ref type_info opAssign_(const ref type_info);
```

Comment:
Is there any reason to declare this at all? It's not virtual...

Reply:
Right that's a leftover (and it wouldn't link). Will remove.